src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php

```diff
@@ -20,7 +20,7 @@
  * This store deals with a "single state" or "multiple state" Marking.
  *
  * "single state" Marking means a subject can be in one and only one state at
- * the same time. Use it with state machine or specific workflow.
+ * the same time. Use it with state machine.
```

Comment:
Either remove
Or need to be more clear on « specific » because it is confusing

Reply:
:+1: